index.html

```diff
@@ -178,5 +178,58 @@ <h4 class="skill-title">Skills</h4>
       </div>
     </div>
   </section>
+  <section class="section-contact-form">
+    <div class="contact-form-container flex gen-margin">
+      <h3 class="contact-form-primary">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+      <form action="https://formspree.io/f/{form_id}" method="POST" class="contact-form">
```

Comment:
- [ ] You did a great job working on the suggested changes, but I'm afraid your form is not submitting because you don't have a real form ID. You are supposed to get a form ID from [Formspree.io](https://formspree.io/) and replace `{form_id}` with the one you get from Formspree. This is what I'm getting when trying to submit the form. ✍️

<img src="https://user-images.githubusercontent.com/17243325/235849649-e4900d81-22a7-43bd-96c7-0138e8384a80.png" width="300">

- [ ] [OPTIONAL] Kindly add `name` attribute on all 3 text inputs so the Formspree service can collect the data properly. 🤝️